tests/neuro_san/client/test_hocons.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from tests.framework.unittest.dynamic_hocon_unit_tests import DynamicHoconUnitTests
+
+
+class TestHocons(TestCase):
+    """
+    Data-driven dynamic test cases where each test case is specified by a single hocon file.
+    """
```

Comment:
An exemplary case of using the DynamicHoconUnitTests class and parameterized.expand annotation
to create dynamic unit tests based on a list of hocon files that fully describe the tests.